### ISSUE TITLE
the region boxes  not match the original images

### ISSUE THREAD

**Issue by liuyyy111** [CrossmodalGroup/GSMN#15]
I found that the region box and size files you provided not match the original images, but it works.
i.e. in the valid dataset, the 23rd image is 1155138244.jpg, and its size is 333 * 500 (h * w), but in the file dev_ims_size.npy, it is 375*500.
How to explain this?

**Comment by liuyyy111** [comment]
The same problem also exists in the training set and test set

**Comment by 184446223** [comment]
the boxes of image region is match the image region feature?
